mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc

```diff
@@ -32,12 +32,12 @@ This step is necessary to instruct API Manager to use your deployed Mule applica
 
 == Configuring Anypoint Platform Organization Credentials in Mule Runtime
 
-To use Autodiscovery in a Mule Application, the Runtime must start with Anypoint Platform credentials already configured. +
+To use Autodiscovery in a Mule application, Mule runtime engine must start with Anypoint Platform credentials already configured. 
```

Comment:
See term reference: "Mule application" and "Mule runtime engine (Mule or Mule runtime)" in the term ref: https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#TermPresentationReference-M

Reply:
I thought we discouraged the use of "+" in paragraphs like this due to the unusual formatting results.

Reply:
```suggestion

To use Autodiscovery in a Mule application, Mule runtime engine must start with Anypoint Platform credentials already configured. 
```

alternative on a separate line